### DISCUSSION TITLE
How to configure SEB 3.3 to allow Zoom running in the background with Moodle

### DISCUSSION THREAD

**Discussion by cienciascontic** [SafeExamBrowser/seb-win-refactoring#158] [Q&A]
Hi!
I would like to know if, in order to allow Zoom running in the background while a student is taking a quiz in Moodle with SEB 3.3, is it enough to just disable the Zoom check box on the "Prohibitted processes" tab in the SEB Configuration utility, or is also needed to set the parameters in the "Permitted processes" like this:
![image](https://user-images.githubusercontent.com/6078186/129432993-6f375754-db6d-43f6-aa44-b4f7e9909fc8.png)

Thanks a lot.
Regards,

Cristián

**Comment by vanie76** [comment]
I set is as below
![image](https://user-images.githubusercontent.com/87995554/129433983-85b7feb9-5e01-419b-97c4-cd35e2a58bd4.png)
![image](https://user-images.githubusercontent.com/87995554/129433992-578dcc86-9913-4b40-9ab2-73aa0580fe1c.png)

**Reply by cienciascontic** [reply]
Hi @vanie76 Did that configuration work for you?

**Reply by vanie76** [reply]
Yes it worked

**Comment by aurelienb1** [comment]
Hi,

When I last tried this feature I noticed a bug.

If you are proctoring with Zoom, each student is in a breakout room with the screen sharing active. If the proctor moves from room to room it breaks the screen sharing when they arrive in the room. This means that the proctor no longer sees what is happening on the screen.

Is this something you have validated with the new version?

Regards,

**Comment by vanie76** [comment]
We have one person in the main room and invigilators in each break up room. We have a backup room where I will stand by there for any issues
We don’t screen share


Vanie

**Comment by vanie76** [comment]
Yes it worked
2nd semester running test with SEB
I only hv issues with macOS

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
